feature/office/src/main/java/com/iteneum/office/presentation/ui/OfficeView.kt

```diff
@@ -71,13 +71,13 @@ fun OfficeUI(
                 ),
                     color = MaterialTheme.colorScheme.tertiary,
                     textAlign = TextAlign.Justify,
-                    weight =FontWeight.Bold ,
+                    weight = FontWeight.Bold,
                     onClick = {})
                 LPBodyMedium(modifier = Modifier.padding(top = sizes.extraSize10),
                     label = officeData.address,
```

Comment:
missing label **Address** in the UI
![image](https://github.com/irv205/droid_leaspert/assets/33376944/7402403d-b77b-4ca0-bc01-0bba598f7df0)

Reply:
word Address is there, but the frame is hiding that part